docs/src/modules/components/ChartsInstallationInstructions.js

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
 import InstallationInstructions from './InstallationInstructions';
 
-// #default-branch-switch
+// #npm-tag-reference
 
 const packages = {
-  Community: '@mui/x-charts',
-  Pro: '@mui/x-charts-pro',
+  Community: '@mui/x-charts@latest',
```

Comment:
IMHO, I think keeping this without a tag on the latest feels cleaner. 🤔

Reply:
thats personal preference i guess ... to me it does not make a real difference, but i felt that this just strengthens the message (if that makes sense 🫣)

Reply:
As far as I've seen, tags are used for scripts to ensure the latest version of the script is used everytime, but for packages, `@latest` is the default whenever you are installing the package, hence, to me it feels like redundant extra noise. 🤔

Reply:
I'm for it, but I'd love to hear @cherniavskii or at least someone else's (cc @mui/x) take as well. 🙈